### PR TITLE
beginnings of NL translation

### PR DESCRIPTION
**Pull request by rscmbbng**
Start of translating Dino to Dutch. Used #33 as a template.

### CODE REVIEW

File: main/po/nl.po [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+"POT-Creation-Date: 2017-04-10 17:36+0200\n"
+"PO-Revision-Date: 2017-04-10 17:50+0200\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.8.11\n"
+"Last-Translator: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: it_IT\n"
```
Comment (svalo) [inline main/po/nl.po]: You might want to update this as well as you did for plugins :)

Comment (rscmbbng) [inline main/po/nl.po]: thanks!

**Comment by fiaxh** [comment]
Thank you for the Durch translation! There is a [weblate](https://hosted.weblate.org/projects/dino/translations/) set up for Dino now, where translations can be edited online. Your translation is already there. The plugin translations are not there yet, but they will be.

**Comment by rscmbbng** [comment]
thanks for integrating the changes!

**Pull request closed without merging**